storage/suffix.go

```diff
@@ -61,11 +61,11 @@ func (s Suffix) String() string {
 }
 
 // ParseSuffix converts a suffix string back into a Suffix.
-func ParseSuffix(s string) (Suffix, error) {
+func ParseSuffix(s string) (*Suffix, error) {
 	b, err := base64.StdEncoding.DecodeString(s)
 	if err != nil {
-		return EmptySuffix, err
+		return &EmptySuffix, err
```

Comment:
this should probably be `nil, err` now - I suspect it was just returning an empty `Suffix` because it couldn't return `nil`.

Reply:
Done. Another mechanical replacement.